core/thread/thread_windows.odin

```diff
@@ -44,6 +37,11 @@ _create :: proc(procedure: Thread_Proc, priority := Thread_Priority.Normal) -> ^
 				runtime.default_temp_allocator_destroy(auto_cast context.temp_allocator.data)
 			}
 		}
+
+		if .Auto_Cleanup in t.flags {
+			free(t, t.creation_allocator)
```

Comment:
Clear the handle here too?

Reply:
`win32_thread` you mean? I only did what `_delete` did on Windows, not sure if that's needed. Perhaps it would be a good idea, idk.

Reply:
Yeah; If you clear it on one, might as well on the other too.

Reply:
Ok let's first see what gB says.